includes/objectcache/MemcachedPeclBagOStuff.php

```diff
@@ -100,7 +104,10 @@ protected function applyDefaultParams( $params ) {
 		}
 
 		if ( !isset( $params['serializer'] ) ) {
-			$params['serializer'] = 'php';
+			// SUS-4749 | use igbinary serializer when available
+			// @see https://github.com/igbinary/igbinary#igbinary
+			// @see https://phpolyk.wordpress.com/2011/08/28/igbinary-the-new-php-serializer/
+			$params['serializer'] = extension_loaded( 'igbinary' ) ? 'igbinary' : 'php';
```

Comment:
are there any machines where we don't have `igbinary` and we fallback to `php`. I wouldn't like to end up in a situation when two machines connecting to the same memcache use different serializers :)